lib/ably/auth.rb

```diff
@@ -82,7 +82,7 @@ def initialize(client, token_params, auth_options)
 
       if has_client_id? && !token_creatable_externally? && !token_option
         raise ArgumentError, 'client_id cannot be provided without a complete API key or means to authenticate. An API key is needed to automatically authenticate with Ably and obtain a token' unless api_key_present?
-        ensure_utf_8 :client_id, client_id
+        client_id = ensure_utf_8 :client_id, client_id
```

Comment:
I realise we don't have Rubocop / style guides defined, and it's down to preference, but our style has been throughout that if a method is being called to obtain a return value, we go with:

```ruby
client_id = ensure_utf_8(:client_id, client_id)
```

Personally I find this a lot easier to read.

Would you mind changing to follow the coding conventions we have used elsewhere please?

Reply:
Sure, I'll patch that in.  I figured that's what was there when I was adding it in so I wasn't sure if there was already a style guide enforcing it.